src/jobs/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Application } from '../domain/interface';
+import logger from '../logger';
+import { Report, ReportItem } from './types';
+
+export const getEmptyReport: () => ReportItem = () => ({
+  count: 0,
+  ids: [],
+  errors: [],
+  errorCount: 0,
+});
+
+export const buildReportItem = (
+  results: PromiseSettledResult<any>[],
+  reportType: keyof Report,
+  jobName: string,
+) => {
+  logger.info(`${jobName} - Building report item for [${reportType}].`);
+  logger.info(`${jobName} - Adding successful requests to [${reportType}] report.`);
+  const resolved = results
+    .filter((res) => res.status === 'fulfilled')
+    .map((success) => {
+      const { value } = success as PromiseFulfilledResult<Application>;
+      logger.info(`${jobName} - Notification succeeded for ${value.appId}`);
```

Comment:
I know I said that you can't go wrong with too much logging... but in this case:
1. when the notification was sent, it was probably logged as sending and success/error message (or should be)
2. This report is being generated for logging purposes, with the success ID listed

All in all I think THIS might be too verbose. If you want it still lets reduce this to a `logger.debug`